.github/workflows/master.yml

```diff
@@ -170,6 +170,54 @@ jobs:
             **/kyuubi-jdbc-engine.log*
             **/kyuubi-hive-sql-engine.log*
 
+  spark-connector-cross-version-test:
+    name: Kyuubi Spark Connector Cross Version Test
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      max-parallel: 6
+      matrix:
+        java:
```

Comment:
might be good to use Java 17 only, Scala version should be considered too.

Reply:
Updated.